src/grafana_agent.py

```diff
@@ -92,6 +92,7 @@ class GrafanaAgentCharm(CharmBase):
     _key_path = "/tmp/agent/grafana-agent.key"
     _ca_path = "/usr/local/share/ca-certificates/grafana-agent-operator.crt"
     _ca_folder_path = "/usr/local/share/ca-certificates"
+    _cloud_ca_path = "/usr/local/share/ca-certificates/cloud-integrator.crt"
```

Comment:
Add same comment proposed in the vm charm PR?

Reply:
Done!